### ISSUE TITLE
Improve searchability with Google Webmaster Tools and XML Sitemap

### ISSUE THREAD

**Issue by vamaraju** [ahoskins/Winston#26]
It would be nice to have the site indexed by Google. You have to use Google Webmaster Tools and possibly create an XML sitemap:

https://www.google.com/webmasters/tools/dashboard
https://www.xml-sitemaps.com/

**Comment by ahoskins** [comment]
Yes.  To my understanding it's sort of a slow chug up the search results.  It makes it a little less trivial to make a site map for a very AJAX heavy site (inherently this way because of Angular), but it seems to be possible according to this blog: http://www.yearofmoo.com/2012/11/angularjs-and-seo.html

I will look into this.
